src/Microsoft.OData.Client/Microsoft.OData.Client.Common.txt

```diff
@@ -176,6 +176,7 @@ ALinq_CantAddQueryOption=Custom query option not allowed.
 ALinq_CantAddDuplicateQueryOption=Can't add duplicate query option '{0}'.
 ALinq_CantAddAstoriaQueryOption=Can't add query option '{0}' because it would conflict with the query options from the translated Linq expression.
 ALinq_CantAddQueryOptionStartingWithDollarSign=Can't add query option '{0}' because it begins with reserved character '$'.
+ALinq_QueryOptionNotSupported=The query option '{0}' is not supported.

```

Comment:
Based on the originally filed issue, this error message still seems just as non-descriptive as the previous one. As the user who filed the original issue this new message still doesn't let me know that the reason I cant use the options is because they are controlled by the server and should not be set by the client.
Maybe something along the lines of: "The query option '{0}' is not supported or is controlled by the odata service"